test/benchmark_http_client_test.cc

```diff
@@ -37,37 +38,81 @@ using namespace testing;
 
 namespace Nighthawk {
 
-class BenchmarkClientTestBase : public Envoy::BaseIntegrationTest,
-                                public TestWithParam<Envoy::Network::Address::IpVersion> {
+class PythonIntegrationTestBase : public TestWithParam<Envoy::Network::Address::IpVersion> {
```

Comment:
Did you consider just running client, Envoy and test server all as independent processes under a  top-level Python script?